changelogs/unreleased/gh-9780-show-active-meta.md

```diff
@@ -0,0 +1,3 @@
+## feature/config
+
+* The `config:info()` functionality has been expanded (gh-9780).
```

Comment:
Nit: As for me, a changelog entry assumes that we point a functionality itself, not just the fact that it is added. For example, 'added active configuration metainformation (usually a revision in a remote source)'.

Reply:
Fixed.